eng/Signing.props

```diff
@@ -23,21 +23,6 @@
     <FileSignInfo Include="apphost.exe;comhost.dll" CertificateName="None" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(SignBinaries)' == 'true'">
-    <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/hostfxr.dll" />
-    <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/hostpolicy.dll" />
-    <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/dotnet.exe" />
-    <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/ijwhost.dll" />
-    <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/winrthost.dll" />
-    <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/nethost.dll" />
-
-    <ItemsToSign Include="$(CrossGenRootPath)**/*.dll" />
```

Comment:
The surrounding line deletes look like good cleanup, but I don't understand this one. Isn't this needed to sign the crossgen'd WindowsDesktop bits?

Reply:
I might have been too aggressive here, as the other lines were duplicated from "core-setup".

But I did search for someone setting `$(SignBinaries)`, and I couldn't find it being used. Did I miss it somewhere?

Reply:
Easy to miss:

https://github.com/dotnet/windowsdesktop/blob/b8893fd3e27cc169ee519629f191dc8336b2b423/signing/Directory.Build.targets#L8-L17

https://github.com/dotnet/windowsdesktop/blob/master/signing/SignBinaries.proj